src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -655,13 +655,13 @@ private boolean addProvisionedSlave(PodTemplate template, Label label) throws Ex
         PodList namedList = client.pods().withLabels(labelsMap).list();
         List<Pod> namedListItems = namedList.getItems();
 
-        if (slaveListItems != null && containerCap < slaveListItems.size()) {
```

Comment:
Thanks! I think this judgement is not correct. if container capacity == slaveListIterms.size(), cloud should not be able to provision slaves any more